gcp_variant_transforms/testing/integration/run_tests_common.py

```diff
@@ -63,18 +63,22 @@ def run(self):
     self._wait_for_all_operations_done()
 
   def _wait_for_all_operations_done(self):
-    """Waits until all operations of ``service`` are done."""
-    time.sleep(60)
+    """Waits until all operations of `service` are done."""
     # pylint: disable=no-member
     operations = self._service.operations()
+    running_operation_names = set()
     for operation_name in self._operation_names:
-      request = operations.get(name=operation_name)
-      response = request.execute()
-      while not response['done']:
-        time.sleep(10)
-        response = request.execute()
-      self._handle_failure(response)
-      self._responses.append(response)
+      running_operation_names.add(operation_name)
```

Comment:
I think you can remove this loop by `running_operation_names = set(_operation_names)`.